samples/apps/bookinfo/route-rule-ratings-test-v2.yaml

```diff
@@ -0,0 +1,17 @@
+type: route-rule
+spec:
+  name: ratings-test-v2
```

Comment:
The main goal of this rule was to be able to write a e2e test where I can use rule(s) to direct traffic to v2/v3 of reviews and ratings(v2).
I can write a new route-rule file which is specific to the tasks and story flow. My preference is to separate that out to a different pr if thats ok with you folks. wdyt?